cg/meta/workflow/nf_analysis.py

```diff
@@ -87,11 +85,13 @@ def get_compute_env(self, case_id: str) -> str:
         """Get the compute environment for the head job based on the case priority."""
         return f"{self.compute_env_base}-{self.get_slurm_qos_for_case(case_id=case_id)}"
 
-    @staticmethod
-    def get_nextflow_config_path(nextflow_config: str | None = None) -> Path | None:
-        """Path to Nextflow config file."""
+    def get_nextflow_config_path(self, case_id: str, nextflow_config: Path | None = None) -> Path:
+        """Path to nextflow config file."""
         if nextflow_config:
             return Path(nextflow_config).absolute()
```

Comment:
it seems that `nextflow_config` is a Path already, so I guess the cast is not necessary, unless the type hint is wrong

Reply:
type hints are wrong. When getting the config path from the CLI is a string. I'll update it. Thanks